lua/autorun/server/ttt2_force_download.lua

```diff
@@ -1,7 +1,7 @@
 -- This file forces clients to download the icons
 -- If you are distributing those files via FastDL, comment out the line below.
 
-if SERVER then
+if engine.ActiveGamemode() == "terrortown" then
```

Comment:
it should still be server only

Reply:
That is ensured, as this file is in the /autorun/server folder

Reply:
my bad